### PR TITLE
[Fixes #7] Add Supervirus (1.0) game and post

### LINKED ISSUE #7: Supervirus - it's working, should be an easy move

### PR DESCRIPTION
**Pull request by samgqroberts**
Fixes #7 

What this PR do?
* updated some information in the README that would have been a little clearer for me as i was setting up my env
* Adds Supervirus game page

Go through the commits, they have messages that say stuff

@shamshirz I'd feel 100% more comfortable about this if you built this locally and lmk what you think. 

Note that this Supervirus build is the result of this bugfix https://github.com/SylverStudios/Supervirus/commit/e28fa87d2b1b1755b53a3e9af97225bc0d0c081e
and the tarball is available in drive here https://drive.google.com/open?id=1EItpGFfcr08zRaITSt5K5dL_LHX051KB

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,25 +17,41 @@ Open a PR, add a new `post` file (Post format: `YEAR-MONTH-DAY-title.MARKUP`). I
...
-```bash
-# macOS rb is 2.2 and insane. Must rbenv
-# https://github.com/rbenv/rbenv/issues/938
+### Ruby Setup
+We need `ruby` installed, at the correct version, to run `jekyll`.
+In order to easily chose our ruby version, we will first install the tool [`rbenv`](https://github.com/rbenv/rbenv#installation).
+(Important Note! You must complete all of the rbenv installation steps, including updating your bash_profile).
 
-rbenv versions
+After install `rbenv`, we will use it to set up the version of ruby required by jekyll, namely ruby 2.6.0:
```
Comment (shamshirz) [inline README.md:30]: So professional, #loveit

File: assets/supervirus/app.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
+(function webpackUniversalModuleDefinition(root, factory) {
```
Comment (shamshirz) [inline assets/supervirus/app.js:1]: muah 💋

File: assets/supervirus/index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+<body>
+  <h2>Supervirus</h2>
+
+  <p>Controls: Arrow Keys.  Hit things smaller than you and avoid things larger than you.  Godspeed.</p>
+
+
+  <div id="cr-stage"></div>
+
+
+  <p class="bottom">Development of game Supervirus by Sylver Studios</p>
```
Comment (shamshirz) [inline assets/supervirus/index.html:43]: 2 words, eh?
> Sylver Studios

File: _posts/2019-02-16-supervirus.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
+---
+layout: supervirus
+title: "Supervirus"
+date: 2019-02-16 12:00:00 -0500
+categories: games
+---
+
+<base href={{ "/assets/supervirus/" | relative_url }}>
```
Comment (shamshirz) [inline _posts/2019-02-16-supervirus.html:8]: vCool. Will certainly need for the future ones

**Review by shamshirz — APPROVED** [review]
Ran it locally, couldn't win. This game is too hard

File: _posts/2019-02-16-supervirus.html (hunk 4 above)
Comment (rjdellecese) [reply _posts/2019-02-16-supervirus.html:8]: Yeah this is str8 🔥 @samgqroberts

**Pull request merged**